### PR TITLE
Created a desktop version branch to implement media query for size 768px and above 

### PR DESCRIPTION
**Pull request by AmitKlohan**
I've a completed the desktop version of my portfolio project and fulfilled the requirements of the project.

- no linters error 
- used correct Github Flow
- I've used best practices in my HTML and CSS
- Add a media query to run for screen size 768px and wider
- Incorporate changes to make this look like the Figma design template as much as possible
- Make uses of  Flexbox and Grid elements

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -344,3 +344,167 @@ header {
   height: 0.5rem;
   align-self: center;
 }
+
+@media only screen and (min-width: 768px) {
+  .head-section {
+    flex-direction: column;
+    background-image: url("desktop_header_bg.png");
```
Comment (zhadier) [inline style.css:355]: - [ ] Kindly add a fixed `max-width` to adjust the size for your logo in the toolbar, right now on some of the larger screen sizes, the logo just gets too big and takes up too much of the screen which does not look as good as it is on some of the smaller screen sizes.

File: style.css [hunk 2]
```diff
@@ -344,3 +344,167 @@ header {
...
+  .card-div {
+    display: flex;
+    flex-direction: column;
+    margin-right: 0;
+  }
+
+  .card4 {
+    flex-direction: row-reverse;
+    order: 3;
+  }
```
Comment (zhadier) [inline style.css:409]: - [ ] One of your cards seems to have a disabled button as you can see below. Kindly enable it so it conforms to the Figma design parameters.

image

File: style.css [hunk 3]
```diff
@@ -344,3 +344,167 @@ header {
...
+    gap: 1rem;
+    width: 45%;
+  }
+
+  .about-myself-second-section {
+    display: flex;
+    flex-direction: column;
+    gap: 1rem;
+    width: 55%;
+  }
```
Comment (zhadier) [inline style.css:453]: - [ ] Kindly add a little padding to the skills images inside your about section so they look similar to the Figma design ( there's a gap between the image and the border )

> Your Design
image

> Figma Design
image

- [ ] Additionally, please also ensure that the headings for your skills have the same color as in the Figma template

File: style.css [hunk 4]
```diff
@@ -344,3 +344,167 @@ header {
...
+    flex-direction: row;
+  }
+
+  .about-mySelf h1 {
+    margin-top: 0;
+  }
+
+  .contact-me {
+    order: 7;
+  }
```
Comment (zhadier) [inline style.css]: - [ ] There seems to be a small gap between your about and contact sections which is not present in the Figma template, kindly remove this gap.

image

File: style.css [hunk 5]
```diff
@@ -344,3 +344,167 @@ header {
...
+
+  form {
+    width: 45rem;
+    margin: 0 auto;
+  }
```
Comment (zhadier) [inline style.css:477]: - [ ] What do you think about styling the `::placeholder` selector in the `textarea` field and introducing a little padding so it looks similar to the Figma design? Right now the placeholder is attached to the border and it doesn't look quite right visually. Please do implement this feature.

image

**Review by zhadier — CHANGES_REQUESTED** [review]
# Hi @AmitKlohan  👋 

Good job so far! 🌤️ 
There are some issues that you still need to work on to go to the next project but you are almost there! 🍓 



## Highlights
✔️ No linter errors
✔️ Github flow was set up correctly
✔️ Professional pull request title and description
✔️ You have done well matching your design to the original Figma design
✔️ You have used CSS grid and flexbox well for aligning the elements

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question @zhadier so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 2 above)
Comment (AmitKlohan) [reply style.css:409]: Done

File: style.css (hunk 3 above)
Comment (AmitKlohan) [reply style.css:453]: done

File: style.css (hunk 4 above)
Comment (AmitKlohan) [reply style.css]: done

File: style.css (hunk 5 above)
Comment (AmitKlohan) [reply style.css:477]: done

File: style.css (hunk 1 above)
Comment (AmitKlohan) [reply style.css:355]: done

**Comment by AmitKlohan** [comment]
Implementation done as suggested above by CR Mr @zhadier 
- Set the logo image size max-width: 8rem.
- Enabled the button.
- Add padding-left: 1rem to skills section.
- Changed the colour as per Figma design.
- Removed the gap between about me and contact us page.

**Review by omarsalem7 — APPROVED** [review]
Hi @AmitKlohan 😊, 
# STATUS: APPROVED! 🥇 💯 

giphy

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
✅ No linter errors. 
✅ Descriptive pull request 
✅ Project has met all the technical aspects you needed for this project. Keep rocking! 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**